cmd/agent/api/agent/agent.go

```diff
@@ -242,3 +244,25 @@ func getTaggerList(w http.ResponseWriter, r *http.Request) {
 	}
 	w.Write(jsonTags)
 }
+
+func getDiscoveredIntegrations(w http.ResponseWriter, r *http.Request) {
+	log.Info("Got a request for discovered integrations. Discovering integrations.")
```

Comment:
is this log line needed?

Reply:
It was for debugging, I can remove it or use `log.Debug` instead